recipes/django-makemessages-xgettext/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "django-makemessages-xgettext" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "d45daec6cd807bf7721d84b6bbf2ddc609ade2e904a8874b49e893bf0172b802" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  noarch: python
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - django >=1.7
+
+test:
+  commands:
+    - python -c "import django; from django.conf import settings; settings.configure(INSTALLED_APPS=['django_makemessages_xgettext']); django.setup(); import django_makemessages_xgettext;"
+
+about:
+  home: https://github.com/resulto/django-makemessages-xgettext
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
```

Comment:
The license file isn't going to be in the source distribution because it's missing from the `MANIFEST.in`.  Please add it manually:

https://conda-forge.org/docs/meta.html#packaging-the-license-manually

Reply:
you're right @synapticarbors ! I was just looking at github repository ... the license is there, but the license is not in the source distribution ... thank you so much.